cmd/config/config.go

```diff
@@ -13,7 +13,9 @@ import (
 
 var (
 	defaultConfig = &Config{}
+	createConfig = &CreateConfig{}
```

Comment:
so you should create the CreateConfig in the `create` command package. And there you can just call it config. Putting it here is indeed confusing

Reply:
ok

Reply:
for example look at this - https://github.com/lyft/flytectl/blob/master/pkg/adminutils/config.go

Reply:
the config gets automatically associated when the right package is loaded

Reply:
I think lets follow the example set by @pmahindrakar-oss, we do not need this right?